ProcessLib/ThermoMechanics/CreateThermoMechanicsProcess.cpp

```diff
@@ -0,0 +1,208 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateThermoMechanicsProcess.h"
+
+#include <cassert>
+
+#include "MaterialLib/SolidModels/CreateLinearElasticIsotropic.h"
+#include "MaterialLib/SolidModels/CreateLubby2.h"
+#include "MaterialLib/SolidModels/CreateEhlers.h"
+#include "ProcessLib/Utils/ParseSecondaryVariables.h"
+
+#include "ThermoMechanicsProcess.h"
+#include "ThermoMechanicsProcessData.h"
+
+namespace ProcessLib
+{
+namespace ThermoMechanics
+{
+template <int DisplacementDim>
+class ThermoMechanicsProcess;
+
+extern template class ThermoMechanicsProcess<2>;
+extern template class ThermoMechanicsProcess<3>;
+
+template <int DisplacementDim>
+std::unique_ptr<Process> createThermoMechanicsProcess(
+    MeshLib::Mesh& mesh,
+    std::unique_ptr<ProcessLib::AbstractJacobianAssembler>&& jacobian_assembler,
+    std::vector<ProcessVariable> const& variables,
+    std::vector<std::unique_ptr<ParameterBase>> const& parameters,
+    unsigned const integration_order,
+    BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{prj__processes__process__type}
+    config.checkConfigParameter("type", "THERMO_MECHANICS");
+    DBUG("Create ThermoMechanicsProcess.");
+
+    // Process variable.
+
+    //! \ogs_file_param{prj__processes__process__THERMO_MECHANICS__process_variables}
+    auto const pv_config = config.getConfigSubtree("process_variables");
+
+    auto process_variables = findProcessVariables(
+        variables, pv_config,
+        {//! \ogs_file_param_special{prj__processes__process__THERMO_MECHANICS__process_variables__temperature}
+         "temperature",
+         //! \ogs_file_param_special{prj__processes__process__THERMO_MECHANICS__process_variables__displacement}
+         "displacement"});
+
+    DBUG("Associate displacement with process variable \'%s\'.",
```

Comment:
Why only debug output for the displacement variable and not for the temperature?